src/extension/provider/.gitignore

```diff
@@ -0,0 +1 @@
+secrets.ts
```

Comment:
Intentional? Same as empty `secrets.ts`, should it be here?

Reply:
It was part of previous secrets sharing with other devs, removing :+1: